cloudnetwork/clientset/versioned/typed/cloudnetwork/v1/cloudnetwork_client.go

```diff
@@ -0,0 +1,3 @@
+// Code generated by client-gen. DO NOT EDIT.
+
+package v1
```

Comment:
@sttts : there's something clearly wrong here...but I couldn't manage to figure out why it doesn't generate this bit. I think there might be something missing in https://github.com/openshift/api/blob/master/cloudnetwork/v1/register.go as it does not have `AddToScheme` like other APIs do (see: https://github.com/openshift/api/blob/master/console/v1/register.go#L22). At the same time: I am not sure that this is what causes this issue.

Reply:
Fixed by PR: https://github.com/openshift/api/pull/892